unitary/examples/quantum_chinese_chess/test_utils.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2023 The Unitary Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import cirq
+
+a0 = cirq.NamedQubit('a0')
```

Comment:
Could we use alpha instead, e.g. `a0 = alpha.QuantumObject('a0')`

Reply:
Thanks Doug. They are now defined as QuantumObjects.